ee/branding/pages/index.tsx

```diff
@@ -1,158 +1,28 @@
-                <span className='label-text'>{t('bui-shared-logo-url')}</span>
-              </label>
-              <input
-                type='url'
-                id='logoUrl'
-                className='input-bordered input'
-                onChange={onChange}
-                value={branding.logoUrl || ''}
-                placeholder='https://company.com/logo.png'
-              />
-              <label className='label'>
-                <span className='label-text-alt'>{t('bui-shared-logo-url-desc')}</span>
-              </label>
-            </div>
-            <div className='form-control w-full md:w-1/2'>
-              <label className='label'>
-                <span className='label-text'>{t('bui-shared-favicon-url')}</span>
-              </label>
-              <input
-                type='url'
-                id='faviconUrl'
-                className='input-bordered input'
-                onChange={onChange}
-                value={branding.faviconUrl || ''}
-                placeholder='https://company.com/favicon.ico'
-              />
-              <label className='label'>
-                <span className='label-text-alt'>{t('bui-shared-favicon-url-desc')}</span>
-              </label>
-            </div>
-            <div className='form-control w-full md:w-1/2'>
-              <label className='label'>
-                <span className='label-text'>{t('branding_company_name_label')}</span>
-              </label>
-              <input
-                type='text'
-                id='companyName'
-                className='input-bordered input'
-                onChange={onChange}
-                value={branding.companyName || ''}
-                placeholder={t('branding_company_name_label')}
-              />
-              <label className='label'>
-                <span className='label-text-alt'>{t('branding_company_name_alt')}</span>
-              </label>
-            </div>
-            <div className='form-control'>
-              <label className='label'>
-                <span className='label-text'>{t('bui-shared-primary-color')}</span>
-              </label>
-              <input
-                type='color'
-                id='primaryColor'
-                onChange={onChange}
-                value={branding.primaryColor || '#25c2a0'}
-              />
-              <label className='label'>
-                <span className='label-text-alt'>{t('bui-shared-primary-color-desc')}</span>
-              </label>
-            </div>
-            <div className='mt-5'>
-              <ButtonPrimary loading={loading}>{t('bui-shared-save-changes')}</ButtonPrimary>
-            </div>
-          </div>
-        </form>
-      </div>
-    </>
+    <BrandingForm
+      defaults={{ primaryColor: '#25c2a0' }}
+      urls={{ getBranding: '/api/admin/branding', post: '/api/admin/branding' }}
+      onUpdate={() => {
+        successToast(t('settings_updated_successfully'));
```

Comment:
All translation strings should be renamed to match our internal-ui convention - `bui-branding-settings-updated-succesfully`

Reply:
This translation is not part of internal-ui as its part of callback that is passed down as prop.

Reply:
The naming convention is just convenience to be able to move it to the other project easily.